mysql-test/suite/sys_vars/t/binlog_space_limit_basic.test

```diff
@@ -0,0 +1,88 @@
+--source include/have_log_bin.inc
+#
+# only global
+#
+select @@global.binlog_space_limit;
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+select @@session.binlog_space_limit;
+show global variables like 'binlog_space_limit';
+show session variables like 'binlog_space_limit';
+--disable_warnings
+select * from information_schema.global_variables where variable_name='binlog_space_limit';
+select * from information_schema.session_variables where variable_name='binlog_space_limit';
+--enable_warnings
+
+#
+# show that it's read-only
+#
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+set global binlog_space_limit=1;
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+set session binlog_space_limit=1;
+
+FLUSH LOGS;
```

Comment:
Please don't mix sys_vars and functional test - keep only the variable properties (only global, read only) in this test, create a new test in binlog suite for the actual functionality